README.md

```diff
@@ -98,3 +98,9 @@ These terms clarify the different types of seeds used in Diablo MapGen and provi
 - **Game Seed:** The time() value set on game creation. It serves as the initial seed for generating dungeon seeds and determines the overall characteristics of a game session.
 - **Dungeon Seed:** The floor-specific seed saved to the seed table. This seed determines the placement of monsters and items within each floor of the dungeon. The dungeon seed is derived from the game seed at the beginning of the game session.
 - **Level Seed:** This is the value that generates the specific layout of tiles for a given level in the dungeon. The level seed is derived by starting with the dungeon seed and then letting the RNG advance each time the level generator does not produce a valid level.
+
+## Additional tools
+
+`seed_table` shows what dungeon seeds correspond to a given game seed and when that game occurred (as game seeds are a unix timestamp representing the local date/time), or show what game seeds lead to a given dungeon seed. This allows expanding an ideal dungeon seed into a complete dungeon table to evaluate the overall run.
```

Comment:
```suggestion
`seed_table` shows what dungeon seeds correspond to a given game seed and when that game occurred (as game seeds are a unix timestamp representing the UTC date/time), or show what game seeds lead to a given dungeon seed. This allows expanding an ideal dungeon seed into a complete dungeon table to evaluate the overall run.
```

Reply:
Unix timestamps are UTC, though there appear to be a bug regarding it in early veresions of the Windows SDK which appears to affect 1.00, but not 1.09 (which we mirror)